### ISSUE TITLE
About Docs and Programmatically changes

### ISSUE THREAD

**Issue by jacksonbenete** [jspreadsheet/ce#1079]
Hello there,
I see that in the Docs there is reference for some functions like jexcel.getColumnNameFromId that can be found in the examples, but this same function isn't anywhere to be found in the "Programmatically changes" section.

I'm wondering if there is many others useful functions, properties or handlers that aren't in the Docs.

You can see the example of what I'm talking about [here](https://bossanova.uk/jexcel/v4/examples/events)
`var changed = function(instance, cell, x, y, value) {
    var cellName = jexcel.getColumnNameFromId([x,y]);
    $('#log').append('New change on cell ' + cellName + ' to: ' + value + '
');
}`

But there is no reference of this ".getColumnNameFromId" to be found in _Programmatically changes_, _Handling event_s nor _Quick reference_ section.

Thanks!

**Comment by pphod** [comment]
Yes, you are right. The helpers functions are not in the documentation yet. But it should be in the future. Thanks ;)
